start-finalSetup02Modpack

```diff
@@ -78,18 +78,26 @@ done
 fi
 
 if [[ "$MANIFEST" ]]; then
-EFFECTIVE_MANIFEST_URL=$(curl -Ls -o /dev/null -w %{url_effective} $MANIFEST)
-case "X$EFFECTIVE_MANIFEST_URL" in
+    if [[ -e "$MANIFEST" ]]; then
+        EFFECTIVE_MANIFEST_FILE=$MANIFEST
+    elif isURL "$MANIFEST"; then
+        EFFECTIVE_MANIFEST_FILE=/tmp/manifest.json
+        curl -Ls -o $EFFECTIVE_MANIFEST_FILE "$MANIFEST"
+    else
+        echo "MANIFEST='$MANIFEST' is not a valid manifest url or location"
```

Comment:
Yeah, I think an exit in this condition makes sense to me too.

Reply:
Added exit in new commit